depends/packages/libxcb_util.mk

```
@@ -0,0 +1,31 @@
+package=libxcb_util
+$(package)_version=0.4.0
+$(package)_download_path=https://xcb.freedesktop.org/dist
+$(package)_file_name=xcb-util-$($(package)_version).tar.bz2
+$(package)_sha256_hash=46e49469cb3b594af1d33176cd7565def2be3fa8be4371d62271fabb5eae50e9
+$(package)_dependencies=libxcb
+
+define $(package)_set_vars
+$(package)_config_opts=--enable-static --disable-devel-docs --without-doxygen
```

For platform plugin dependencies, the `QXcbIntegrationPlugin` is such a plugin, no static linkage required:
```suggestion
$(package)_config_opts=--disable-static --disable-devel-docs --without-doxygen
```

---

Done.